src/plugins/kibana/public/kibana.js

```diff
@@ -12,6 +12,8 @@ var modules = require('ui/modules');
 
 var kibanaLogoUrl = require('ui/images/kibana.png');
 
+routes.start();
```

Comment:
Good point, I'll update the PR. I can also create PRs for the others plugins that needs to enable the router. Marvel, Timelion, others?